src/main/scala/djinni/CppCliMarshal.scala

```diff
@@ -27,7 +27,7 @@ class CppCliMarshal(spec: Spec) extends Marshal(spec) {
   def typename(ty: TypeRef, needsHandle: Boolean): String = toCppCliType(ty, None, Seq(), needsHandle)
   def typename(name: String, ty: TypeDef): String = idCs.ty(name)
 
-  override def fqTypename(tm: meta.MExpr): String = toCppCliType(tm, Some(spec.cppCliNamespace.replace(".", "::")), Seq(), needsHandle = true)
+  override def fqTypename(tm: meta.MExpr): String = toCppCliType(tm, Some(spec.cppCliNamespace), Seq(), needsHandle = true)
```

Comment:
I'd like to discuss if the replacing is required/appropriate here.
- While not explicitly stated in the docs, my understanding is that `--cppcli-namespace` should be a C++-style namespace, so the replacement is not necessary?
- If we decided to do namespace conversion to also allow C# style namespaces, I think the conversion should happen in `Main.scala` before being set on the `Spec`?

Reply:
I think the replacement made sense with the original flag: `--cs-namespace`. Now I agree that it's somewhat redundant.

Reply:
Just a question, a C++ namespace can not contain a `.` 
So if you have the replacement, wouldn't it be up to the user to either take a C# pr C++ namespace, and both would just work?

Reply:
If we do the namespace replacement in-place just here, I am not sure it will work. There may be more places that would need the namespace to be converted. 🤔 
I think I removed the C#-namespace support when converting the CLI parameters from `--cs-*` to `--cppcli-*`.
If we decided to support both namespace types, we'd need to check that, I am not sure about it.

Reply:
Thanks for info.
If it's complicated, do not do it in this context.
If it's needed, we make a new ticket and take that extra